config/jobs/tarmak/tarmak-presubmits.yaml

```diff
@@ -129,133 +79,7 @@
   trigger: "(?m)^/test( all| acceptance),?(\\s+|$)"
   rerun_command: "/test acceptance"
```

Comment:
I think the trigger and rerun command here should be removed, in order to force users of this template to set them explicitly?

Reply:
Makes sense